lib/communications/ActuatorArray.cpp

```diff
@@ -0,0 +1,80 @@
+
+#include "actuator.h"
+#include <cstdlib>
+#include <Arduino.h>
+
+
+int8_t binary_search(uint8_t *array, uint8_t start, uint8_t end, uint8_t val);
+int8_t sequential_search(uint8_t *array, uint8_t size, uint8_t val);
+
+ActuatorArray::ActuatorArray(const uint8_t size) {
+  _backingStore = (Actuator **) std::malloc(size * sizeof(Actuator *));
```

Comment:
if we have a set `size` number of Actuators, can't this just be a static array that is created in the highest level stack frame and passed down as needed? To avoid needing to use malloc, since I imagine this thing is intended to be created once at the beginning & used as-is for the rest of the execution of the code